src/main/java/com/redhat/devtools/intellij/quarkus/lsp/QuarkusLanguageClient.java

```diff
@@ -74,6 +87,24 @@ private void sendPropertiesChangeEvent(List<MicroProfilePropertiesScope> scope,
         }
     }
 
+    @Override
+    public void profileChanged(@NotNull InspectionProfile profile) {
+        MicroProfileInspectionsInfo newInspectionState = MicroProfileInspectionsInfo.getMicroProfileInspectionInfo(getProject());
```

Comment:
Explain that you are tracking MP validtaion settings (declared in Editor/Inspection/MicroProfue UI settings) to push config to the LS with didChangeConfig.

Reply:
done